### PR TITLE
test: update logcollector to get secrets -oyaml

### PR DESCRIPTION
**Pull request by subhamkrai**
in case of secrets we need to get `-oyaml` instead
of `describe` since in describe it will be hidden.

Signed-off-by: subhamkrai <srai@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: tests/scripts/collect-logs.sh [hunk 1]
```diff
@@ -27,32 +27,36 @@ for NAMESPACE in "${NAMESPACES[@]}"; do
...
       # collect logs for pods along the way
       if [[ "$KIND" == 'pod' ]]; then
-        kubectl -n "$NAMESPACE" logs --all-containers "$resource" > "${NS_DIR}"/logs--"$resource".txt
+        kubectl -n "$NAMESPACE" logs --all-containers "$resource" >"${NS_DIR}"/logs--"$resource".txt
       fi
     done
   done
 
+  for secret in $(kubectl -n "$NAMESPACE" get secrets -o jsonpath='{.items[*].metadata.name}'); do
+    kubectl -n "$NAMESPACE" get -o yaml secret "$secret" >"${NS_DIR}"/secret-describe--"$secret".txt
```
Comment (BlaineEXE) [inline tests/scripts/collect-logs.sh:43]:
```suggestion
    kubectl -n "$NAMESPACE" get -o yaml secret "$secret" > "${NS_DIR}"/secret-describe--"$secret".txt
```

**Review by BlaineEXE — APPROVED** [review]
Two minor comments. It looks good to merge once you change those quickly.

File: tests/scripts/collect-logs.sh (hunk 1 above)
Comment (subhamkrai) [reply tests/scripts/collect-logs.sh:45]: my vscode auto-save removing the space so keeping it, let me know if we need a space, I see vscode adding space before `>` but not after `>`

File: tests/scripts/collect-logs.sh [hunk 2]
```diff
@@ -27,32 +27,36 @@ for NAMESPACE in "${NAMESPACES[@]}"; do
...
 
       # collect logs for pods along the way
       if [[ "$KIND" == 'pod' ]]; then
-        kubectl -n "$NAMESPACE" logs --all-containers "$resource" > "${NS_DIR}"/logs--"$resource".txt
+        kubectl -n "$NAMESPACE" logs --all-containers "$resource" >"${NS_DIR}"/logs--"$resource".txt
       fi
     done
   done
 
+  for secret in $(kubectl -n "$NAMESPACE" get secrets -o jsonpath='{.items[*].metadata.name}'); do
```
[the remark this answers is not in the document]
Comment (subhamkrai) [reply tests/scripts/collect-logs.sh:44]: added

File: tests/framework/installer/ceph_settings.go [hunk 3]
```diff
@@ -85,6 +85,7 @@ func (s *TestCephSettings) readManifestFromGitHubWithClusterNamespace(filename,
 }
 
 func (s *TestCephSettings) replaceOperatorSettings(manifest string) string {
+	manifest = strings.ReplaceAll(manifest, `ROOK_LOG_LEVEL: "INFO"`, `ROOK_LOG_LEVEL: "DEBUG"`)
```
Comment (travisn) [inline tests/framework/installer/ceph_settings.go:88]: I'm a bit hesitant to always enable the debug logs in all the tests, but let's try it for now

**Review by travisn — APPROVED** [review]

**Pull request merged**